### DISCUSSION TITLE
How do I keep the speaker of the original model when fine-tuning

### DISCUSSION THREAD

**Discussion by pkuhsczy** [coqui-ai/TTS#3042] [General Q&A]
When I was fine-tuning the vctk corresponding to the multi-speaker version of vits, I found that the speaker in my training set overwrote the speaker in the original model corresponding to vctk. How do I keep them?

**Comment by eginhard** [comment]
The speakers in the public `tts_models/en/vctk/vits` model are randomly shuffled, see #2258, so further fine-tuning with the original VCTK data would be difficult.

**Reply by pkuhsczy** [reply]
Thank you very much for your answer. What I need now is equivalent to adding a speaker of my own data set to the original model. But I found that after the fine-tuning, the 100 + speakers in tts_models/en/vctk/vits were gone, and only my own was left. I don't know how to fix this.
